core/templates/dev/head/pages/exploration_player/AudioControlsDirective.js

```diff
@@ -53,39 +56,56 @@ oppia.directive('audioControls', [
               AudioTranslationManagerService.getCurrentAudioLanguageCode()];
           };
 
-          $scope.AudioPlayerService = AudioPlayerService;
+          $scope.isAudioPlaying = function() {
+            return AudioPlayerService.isPlaying() ||
+              AutogeneratedAudioPlayerService.isPlaying();
+          };
 
           $scope.IMAGE_URL_REWIND_AUDIO_BUTTON = (
             UrlInterpolationService.getStaticImageUrl(
               '/icons/rewind-five.svg'));
 
           $scope.isAudioAvailableInCurrentLanguage = function() {
-            return Boolean(getAudioTranslationInCurrentLanguage());
+            return Boolean(getAudioTranslationInCurrentLanguage()) ||
+              AudioTranslationManagerService
+                .isAutogeneratedLanguageCodeSelected();
```

Comment:
Hm, might want a helper function locally here for checking isAutogeneratedLanguageCodeSelected(). The length of this whole thing seems to affect the readability of not only this line, but below too.

Reply:
done